packages/amplify-e2e-core/src/utils/pinpoint.ts

```diff
@@ -38,6 +38,7 @@ const serviceRegionMap = {
   'ap-northeast-1': 'ap-northeast-1',
   'eu-central-1': 'eu-central-1',
   'eu-north-1': 'eu-central-1',
+  'eu-south-1': 'eu-central-1',
```

Comment:
Pinpoint is not available in eu-south-1, so we fall back to central 

https://docs.aws.amazon.com/general/latest/gr/pinpoint.html